great-idea/index.html

```diff
@@ -17,7 +17,83 @@
   <body>
 
     <!-- Copy and paste your HTML from the first UI project here -->
-      
+    <div class="container">
+
+    
+      <header>
+         <nav>
+           <a href=services.html>Services</a>
+           <a href="#">Product</a>
+           <a href="#">Vision</a>
+           <a href="#">Features</a>
+           <a href="#">About</a>
+           <a href="#">Contact</a>
+         </nav>
+         <img class="logo" src="https://tk-assets.lambdaschool.com/c7e8c14d-d7ad-4d21-9c18-4a68c77e5ad5_logo.png" alt="Great Idea! Company logo.">
+       </header>
+     
+     <section class="top-content">
+       <div class="tc-text">
+         <h1>
+           Innovation <br> On <br> Demand
+         </h1>
+         <button>Get Started</button> 
+       </div>
+       <img src="img/header-img.png" alt="Image of a code snippet.">
+     </section>
+     
+     <section class="main-content">
+     
+      <div class="first-sec">
+     
+         <div class="text-content">
+       <h2>Features</h2>
+       Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+         </div>
+     
+         <div class="text-content">
+       <h2>About</h2>
+       Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+         </div>
+     
+     </div>
+     
+       <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+       
+       <div class="second-sec">
+     
+           <div class="text-content">
+         <h2>Services</h2>
+         Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
```

Comment:
I think it'd be best to wrap this text around a `p` tag, so that you can target this text if you need to style it in the CSS.